assets/stylesheets/shared/_extends-forms.scss

```diff
@@ -1,8 +1,4 @@
-/**
- * SASS placeholder selectors for form styles
- *
- * @format
```

Comment:
Can we keep the prettier pragma?

Reply:
The pragma needs to be in a `/**`-ish comment. But these are then shipped in the production bundle. The change to `//` tries to avoid that. I don't know if there is a way to have both -- pragma that gets removed when bundling for production. Seems there isn't.